website/content/en/docs/ansible/reference/advanced_options.md

```diff
@@ -212,12 +212,9 @@ spec:
 
 ## Passing Arbitrary Arguments to Ansible
 
-Users can specify an arbitrary command-line argument to the ```operator-sdk exec-entrypoint``` and ```run --local``` subcommands for Ansible-based Operators via the ```ansible-args``` flag.
+You are able to use the flag `--ansible-args` to pass an arbitrary argument to the Ansible-based Operator. With this option we can, for example, allow a playbook to run a specific part of the configuration without running the whole playbook:  
```

Comment:
A second thought on this ... The so called `--ansible-args` are actually the arguments to be passed to `ansible-runner` rather than `ansible` or `ansible-playbook`. This means we can pass anything below using this new flag:

```
usage: ansible-runner [-h] [--version] [-p PLAYBOOK | -m MODULE | -r ROLE]
                      [--debug] [--logfile LOGFILE] [-b BINARY] [-i IDENT]
                      [--rotate-artifacts ROTATE_ARTIFACTS]
                      [--artifact-dir ARTIFACT_DIR]
                      [--project-dir PROJECT_DIR] [--inventory INVENTORY] [-j]
                      [--omit-event-data] [--only-failed-event-data] [-q] [-v]
                      [--limit LIMIT] [--cmdline CMDLINE] [--hosts HOSTS]
                      [--forks FORKS] [--roles-path ROLES_PATH]
                      [--role-vars ROLE_VARS] [--role-skip-facts]
                      [-a MODULE_ARGS] [--process-isolation]
                      [--process-isolation-executable PROCESS_ISOLATION_EXECUTABLE]
                      [--process-isolation-path PROCESS_ISOLATION_PATH]
                      [--process-isolation-hide-paths [PROCESS_ISOLATION_HIDE_PATHS [PROCESS_ISOLATION_HIDE_PATHS ...]]]
                      [--process-isolation-show-paths [PROCESS_ISOLATION_SHOW_PATHS [PROCESS_ISOLATION_SHOW_PATHS ...]]]
                      [--process-isolation-ro-paths [PROCESS_ISOLATION_RO_PATHS [PROCESS_ISOLATION_RO_PATHS ...]]]
                      [--directory-isolation-base-path DIRECTORY_ISOLATION_BASE_PATH]
                      [--resource-profiling]
                      [--resource-profiling-base-cgroup RESOURCE_PROFILING_BASE_CGROUP]
                      [--resource-profiling-cpu-poll-interval RESOURCE_PROFILING_CPU_POLL_INTERVAL]
                      [--resource-profiling-memory-poll-interval RESOURCE_PROFILING_MEMORY_POLL_INTERVAL]
                      [--resource-profiling-pid-poll-interval RESOURCE_PROFILING_PID_POLL_INTERVAL]
                      [--resource-profiling-results-dir RESOURCE_PROFILING_RESULTS_DIR]
                      COMMAND private_data_dir
```

It would be awkward if we later want to pass more arguments to `ansible` if we call this flag `--ansible-args`. Not quite sure will be exposing the `--cmdline CMDLINE` someday or even we do have such use cases. However, just for avoiding confusion, would it be better to call this as `--runner-args`?

Reply:
These are actually arguments to ansible, not runner

Reply:
They are written to the `env/cmdline` directory and then ansible-runner passes them to the `ansible` command that it runs

Reply:
Ah, didn't read the code carefully. That solves my concern. Thank you, @fabianvf .